slick/src/main/scala/slick/lifted/Query.scala

```diff
@@ -24,6 +24,12 @@ sealed abstract class Query[+E, U, C[_]] extends QueryBase[C[U]] { self =>
   def shaped: ShapedValue[_ <: E, U]
   final lazy val packed = shaped.toNode
 
+  /** Add a comment on top of the query
+    * */
+  def withComments(comment: String): Query[E, U, C] = {
+    new WrappingQuery[E, U, C](new Comment(comment, toNode), shaped)
+  }
```

Comment:
Should this be `withComment` (singular)? And should we add a `comment` method to be consistent with `filter` and `withFilter`?